spec/system/admin_flags_spec.rb

```diff
@@ -147,6 +147,27 @@
     )
   end
 
+  it "has settings tab" do
+    admin_flags_page.visit
+    expect(page.all(".admin-nav-submenu__tabs a").map(&:text)).to eq(%w[Settings Flags])
```

Comment:
Should use I18n for the tab names. The list of settings below doesn't need it because we don't translate settings. Also could be nice to have a page object component for the header, then you could do `expect(admin_header).to have_tabs([x,y])`